File: 백준/Bronze/2566. 최댓값/최댓값.py
# 최댓값
A = []
for i in range(9):
    A.append(list(map(int,input().split())))


max_num = 0
max_i = 0
max_j =0

for i in range(9):
    for j in range(9):
        if max_num <= A[i][j]        :
            max_num = A[i][j]
            max_i = i+1
            max_j = j+1
print(max_num,max_i,max_j)

File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[][] arr = new int[9][9];

        for (int i = 0; i < 9; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            for (int j = 0; j < 9; j++) {
                arr[i][j] = Integer.parseInt(st.nextToken());
            }
        }
        int max = 0;
        int max_i = 0;
        int max_j = 0;
        for (int i = 0; i < 9; i++) {
            for (int j = 0; j < 9; j++) {
                if (arr[i][j] > max) {
                    max = arr[i][j];
                    max_i = i;
                    max_j = j;
                }
            }
        }
        System.out.println(max);
        System.out.print((max_i+1)+" ");
        System.out.print((max_j+1));

    }
}
